wagtail_transfer/wagtail_hooks.py

```diff
@@ -23,3 +25,9 @@ def is_shown(self, request):
 @hooks.register('register_admin_menu_item')
 def register_admin_menu_item():
     return WagtailTransferMenuItem('Import pages', reverse('wagtail_transfer_admin:choose_page'), order=10000)
+
+
+# Register a custom css file for the wagtail admin.
+@hooks.register("insert_global_admin_css", order=1000)
+def global_admin_css():
```

Comment:
This file is only needed on the choose page, so better to add it into the template instead I think

https://github.com/wagtail/wagtail-transfer/blob/master/wagtail_transfer/templates/wagtail_transfer/choose_page.html#L4-L8

Reply:
Yeah - I did it this way initially because I was thinking of putting a new icon in for the menu, but you're right, it's not needed yet.

Reply:
Fixed now!